dask/multiprocessing.py

```diff
@@ -125,6 +126,23 @@ def pack_exception(e, dumps):
     return result
 
 
+_CONTEXT_UNSUPPORTED = """\
+The 'multiprocessing.context' configuration option will be ignored on Python 2
+and on Windows, because they each only support a single context.
+"""
+
+
+def get_context():
+    """ Return the current multiprocessing context."""
+    if sys.platform == "win32" or sys.version_info.major == 2:
+        # Just do the default, since we can't change it:
+        if config.get("multiprocessing.context", None) is not None:
+            warn(_CONTEXT_UNSUPPORTED, UserWarning)
```

Comment:
I suspect that this will become annoying for users that switch between Python 2 and 3 and want to specify a multiprocessing context in Python 3.  I appreciate that this concern is probably in direct contraction with the "lets warn users if we're doing something silently" concern, but I still anticipate that we'll get some confusion about it.  

Does anyone have thoughts on what the right thing to do here is?

Reply:
1. People who set this option know what they're doing, probably, so are less likely to be confused.
2. Given better default (the follow up ticket), less people will be interested in setting this.

Reply:
What about just erroring? It's not supported on Python 2. So we probably shouldn't let people set this there.